src/server/utils/Database.ts

```diff
@@ -14,4 +13,6 @@ provider.connect().catch((err) => {
   process.exit(1);
 });
 
+// TODO: check if old config exists and tell user about migration path
```

Comment:
I planned to migrate the old config in the setup step

Reply:
makes sense. would be a good option